packages/core/src/Coordinator.js

```diff
@@ -134,18 +134,20 @@ export class Coordinator {
    *  or a SQL string.
    * @param {object} [options] An options object.
    * @param {'arrow' | 'json'} [options.type] The query result format type.
-   * @param {boolean} [options.cache=true] If true, cache the query result.
+   * @param {boolean} [options.cache=true] If true, cache the query result in the client (QueryManager).
+   * @param {boolean} [options.persist=false] If true, persist cached query result in the server.
```

Comment:
`persist` option was missing in the type definition causing an error

Reply:
> Can you say more about where the error was coming from?

The type error was from [`{ persist: true }` in`getFieldInfo`](https://github.com/uwdata/mosaic/blob/71cf64e074ab3c72401f599fd256317eec26cd3b/packages/core/src/util/field-info.js#L67). Currently, there is `persist` field in the `options` type info of `Coordinator.query` here.

> I believe persist should be part of the aggregated ...options, not broken out individually.

Yes, `persist` type info is added as a member of `options` object (`options.persist` not `persist`), not as a separate field. Do you want to add a new interface type definition for `options` separately?

Reply:
I now see what you mean. Updated the code.